src/DotNetBridge/NativeDataView.cs

```diff
@@ -310,13 +308,13 @@ public ValueGetter<TValue> GetGetter<TValue>(int col)
                         };
                 }
 
-                public bool IsColumnActive(int col)
+                public override bool IsColumnActive(int col)
                 {
-                    Contracts.Check(0 <= col && col < Schema.ColumnCount);
+                    Contracts.Check(0 <= col && col < Schema.Count);
                     return _active[col];
                 }
 
-                public override void Dispose()
+                public new void Dispose()
```

Comment:
>new [](start = 23, length = 3)

Instead of this, do:
protected override void Dispose(bool disposing)
{
...
base.Dispose(disposing);
}

Reply:
fixed

---
In reply to: [248381704](https://github.com/Microsoft/NimbusML/pull/77#discussion_r248381704) [](ancestors = 248381704)